tests/test_transforms.py

```diff
@@ -62,17 +64,17 @@ def test_q_to_tth_bad(wavelength, q, expected_error_type, invalid_q_or_d_or_wave
 @pytest.mark.parametrize(
     "wavelength, tth, expected_q",
     [
-        # UC0: user specified empty tth values (without wavelength)
+        # C0: Empty tth values (without wavelength)
         (None, np.array([]), np.array([])),
-        # UC1: user specified empty tth values (with wavelength)
+        # C1: Empty tth values (with wavelength)
         (4 * np.pi, np.array([]), np.array([])),
-        # UC2: user specified valid tth values between 0-180 degrees (without wavelength)
+        # C2: valid tth values between 0-180 degrees (without wavelength)
         (
             None,
             np.array([0, 30, 60, 90, 120, 180]),
             np.array([0, 1, 2, 3, 4, 5]),
         ),
-        # UC3: user specified valid tth values between 0-180 degrees (with wavelength)
+        # C3: valid tth values between 0-180 degrees (with wavelength)
```

Comment:
I don't think we want to call it `UC` right? Since this is testing `actual_q = tth_to_q(tth, wavelength)`

Reply:
We basically never want to see UCs in tests.  UCs are enumerated lists of actions that describe user behavior when using an app.